streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -96,13 +101,16 @@ public void testReprocessingFromScratchAfterReset() throws Exception {
         final KeyValue<Object, Object> result2 = IntegrationTestUtils.waitUntilMinKeyValueRecordsReceived(resultTopicConsumerConfig, OUTPUT_TOPIC_2, 1).get(0);
 
         streams.close();
+        TestUtils.waitForCondition(this.consumerGroupInactive, 5 * STREAMS_CONSUMER_TIMEOUT,
```

Comment:
Well. In case we would change the consumer timeout, we would need to adjust this value too, if we use 10000 directly. It should indicate, that we wait max 5 times the consumer timeout -- IMHO, using 10000 does not make clear where the value comes from. But if you insist, I can also change it.